README.md

```diff
@@ -1,4 +1,20 @@
-# Python boilerplate for GitHub tasks
+# Products out of date
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start
-- Implement the task described [here](app/main.py)
+
+Write tests for `outdated_products` function, that takes dictionary
+`products` and returns list of names of products that are out 
+of date (expiration
+date < today date). Argument `products` is dictionary, where keys are
+names of the products and values are their expiration dates.
+```python
+print(outdated_products(
+    {
```

Comment:
Maybe it is more interesting to create a list of products where each product is a dict? 
Example:
```python
[
  {
    "name": "chicken",
    "expiration_date":  datetime.date(2022, 2, 5),
    "price": 123,
  },
  {
    "name": "salmon",
    "expiration_date":  datetime.date(2022, 2, 10),
    "price": 123,
  },
]
```